Lib/fontTools/subset/__init__.py

```diff
@@ -2858,13 +2862,7 @@ def _subset_glyphs(self, font):
 				del font[tag]
 
 		with timer("subset GlyphOrder"):
```

Comment:
Maybe it doesn't matter but it seems like this is now timing something very different than before?

Reply:
yeah, there wasn't much to time even before the change, but I'll keep it there.